### ISSUE TITLE
Always use comma notation for subsetting data frame columns

### ISSUE THREAD

**Issue by jsta** [datacarpentry/r-intro-geospatial#88]
Lines [L315:L325](https://github.com/datacarpentry/r-intro-geospatial/blob/5ff26f427c2d4b57ee6d0d0eceeb700419191e7a/_episodes_rmd/05-data-subsetting.Rmd#L315:L325) in Lesson 5 advocate for subsetting columns without comma notation:

`head(gapminder[3])`

I'd argue that for clarity sake, we should change this to always use comma notation:

`head(gapminder[,3]`

See #86

**Comment by albhasan** [comment]
I disagree with this suggestion because the objects returned by each operation are different:
```
> class(gapminder[3]) == class(gapminder[ ,3])
[1] FALSE
>
> mode(gapminder[3]) == mode(gapminder[ ,3])
[1] FALSE
>
> typeof(gapminder[3]) == typeof(gapminder[ ,3])
[1] FALSE
```

However, both objects contain the same values: 
```
> all(gapminder[3] == gapminder[ ,3])
[1] TRUE
```

For beginners, or even experts, this subtle change could go unnoticed and produce unexpected consequences. 

For that reason, I think it's better to close this issue without implementing the suggested changes.

**Comment by coopermkr** [comment]
Agreed, and given the age of the issue I'm going to close now.
